Carwash/order_details/views.py

```diff
@@ -1,3 +1,17 @@
-from django.shortcuts import render
+from rest_framework import viewsets
+from rest_framework import permissions
 
-# Create your views here.
+from order_details.serializer import OrderDetailSerializer
+from order_details.models import OrderDetail
+
+
+class OrderDetailViewSet(viewsets.ModelViewSet):
+    """
+    API endpoint that allows OrderDetail to be viewed or edited.
+    """
+    queryset = OrderDetail.objects.all().order_by('order')
+    serializer_class = OrderDetailSerializer
+    permission_classes = [permissions.IsAuthenticated]
+
+    def perform_create(self, serializer):
```

Comment:
why do you add this method?

Reply:
Sorry, this is a mistake. It was prom REST_FRAMEWORK tutorial. I'll remove it.